lib/Chat/Parser/SystemMessage.php

```diff
@@ -563,12 +566,14 @@ public function parseDeletedMessage(Message $chatMessage): void {
 		$parsedParameters = ['actor' => $this->getActor($room, $data['deleted_by_type'], $data['deleted_by_id'])];
 
 		$participant = $chatMessage->getParticipant();
-		$currentActorId = $participant->getAttendee()->getActorId();
+		$currentActorId = $participant?->getAttendee()->getActorId();
 
 		$authorIsActor = $data['deleted_by_type'] === $chatMessage->getComment()->getActorType()
 			&& $data['deleted_by_id'] === $chatMessage->getComment()->getActorId();
 
-		if (!$participant->isGuest()) {
+		if ($participant === null) {
+			$currentUserIsActor = false;
+		} elseif (!$participant->isGuest()) {
 			$currentUserIsActor = $parsedParameters['actor']['type'] === 'user' &&
 				$participant->getAttendee()->getActorType() === Attendee::ACTOR_USERS &&
 				$currentActorId === $parsedParameters['actor']['id'];
```

Comment:
Nit: since `$currentActorId` is not used if `$participant` is null. I think it could be removed from line 569 and replaced by this
```suggestion
				$participant->getAttendee()->getActorId() === $parsedParameters['actor']['id'];
```
(same for line 583)